.github/workflows/ci.yml

```diff
@@ -558,6 +579,9 @@ jobs:
           -D maven.javadoc.skip=true
           -D skipUTs -D skipChecks
           -D failsafe.rerunFailingTestsCount=3 -D flaky.test.reportDir=failsafe-reports
+          -Dio.camunda.process.test.camundaDockerImageName="${{ env.CAMUNDA_DOCKER_IMAGE_NAME }}"
+          -Dio.camunda.process.test.camundaDockerImageVersion="${{ env.DOCKER_IMAGE_TAG }}"
+          -Dio.camunda.process.test.camundaVersion="${{ env.DOCKER_IMAGE_TAG }}"
```

Comment:
💭 I assume the presence of those `io.camunda.process.test.*` variables in the other matrix builds does not influence/confuse them?

Reply:
Correct. These parameters are specific to the CPT modules. They don't influence other modules. :+1: